runtime/dolphin/src/lib.rs

```diff
@@ -825,6 +825,7 @@ mod benches {
         [pallet_asset_manager, AssetManager]
         // Nimbus pallets
         [pallet_author_inherent, AuthorInherent]
+
```

Comment:
nit: can get rid of this to reduce the diff

Reply:
done